modules/pool/lib/pool-gql-loader.service.ts

```diff
@@ -84,10 +87,11 @@ export class PoolGqlLoaderService {
         );
     }
 
-    public async getPools(args: QueryPoolGetPoolsArgs): Promise<GqlPoolMinimal[]> {
+    public async getPools(args: QueryPoolGetPoolsArgs, paths: string[] = []): Promise<GqlPoolMinimal[]> {
         // only include wallet and staked balances if the query requests it
         // this makes sure that we don't load ALL user balances when we don't filter on userAddress
         // need to support ordering and paging by userbalanceUsd. Need to take care of that here, as the DB does not (and should not) store the usd balance
+        // TODO: let's find a way to do that in the DB
```

Comment:
I dont think we can for the simple fact that we dont store userBalances in USD (and for good reason)

I think the gql-loader service would need a general refactor.